main/gen_gdl90.go

```diff
@@ -1046,6 +1047,7 @@ func defaultSettings() {
 	globalSettings.UAT_Enabled = true
 	globalSettings.ES_Enabled = true
 	globalSettings.GPS_Enabled = true
+	globalSettings.GpsdAddress = "localhost:2947"
```

Comment:
Should be fine to use a constant, this is something that will rarely (never) be changed in production.